content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -625,9 +626,11 @@ When the Datadog SDK is configured with the Datadog format for extraction or inj
 
 When the Datadog SDK is configured with the None format for extraction or injection (possibly both), the Datadog SDK does _not_ interact with request headers, meaning that the corresponding context propagation operation does nothing.
 
-### Baggage format
+### Baggage
+
+_Currently available in Python and Node.js. For other languages, please reach out to [Support][11]_ 
 
-Baggage propagation through OpenTelemetry's W3C-compatible headers is enabled by default.
+By default, Baggage is automatically propagated through a distributed request using OpenTelemetry's [W3C-compatible headers][10].
```

Comment:
Should we say that the default propagation style is now "datadog,tracecontext,baggage"?
Users can also disable the baggage propagator by explicitly configuring the settings above and not including the baggage propagator (ie "datadog,tracecontext"). Just a thought --> no need to add it if you think it's not needed or if it's stated somewhere else.

Reply:
That's included at the top of the page, so probably fine to omit here
<img width="769" alt="image" src="https://github.com/user-attachments/assets/c1fc97d9-76b7-444b-9267-aa604b0f0725">

I'll add a note that it can be disabled though